package.json

```diff
@@ -38,6 +38,7 @@
     "eslint": "4.5.0",
     "eslint-plugin-prettier": "2.2.0",
     "husky": "0.14.3",
+    "is-promise": "2.1.0",
```

Comment:
This PR looks good but pulling in a dependency for this check feels like overkill to me. Could you replace this usage of `is-promise` with a function that just checks for the existence of a `then` method?